daemons/dss-checkout-sfn/app.py

```diff
@@ -0,0 +1,96 @@
+import os
+import sys
+
+import domovoi
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'domovoilib'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss
+from dss.util.state_machine.checkout_states import state_machine_def
+from dss.util.email import send_checkout_success_email, send_checkout_failure_email
+from dss.util.checkout import (parallel_copy, get_dst_bundle_prefix, get_manifest_files,
+                               validate_file_dst, validate)
+
+app = domovoi.Domovoi()
+dss.Config.set_config(dss.BucketConfig.NORMAL)
+
+replica = "aws"
+
+logger = dss.get_logger()
+email_sender = dss.Config.get_notification_email()
+default_checkout_bucket = dss.Config.get_s3_checkout_bucket()
+
+@app.step_function_task(state_name="ScheduleCopy", state_machine_definition=state_machine_def)
+def worker(event, context):
+    bundle_id = event["bundle"]
+    version = event["version"]
+    dss_bucket = event["dss_bucket"]
+    dst_bucket = get_dst_bucket(event)
+
+    scheduled = 0
+    for src_key, dst_key in get_manifest_files(bundle_id, version, replica):
+        logger.debug("Copying a file " + dst_key)
+        parallel_copy(dss_bucket, src_key, dst_bucket, dst_key)
+        scheduled += 1
+    return {"files_scheduled": scheduled, "dst_location": get_dst_bundle_prefix(bundle_id, version), "wait_time": 5}
```

Comment:
5 = magic number alert.

also, as a general practice, it's better to give units to variables associated with time, i.e., `wait_time_seconds`

Reply:
This controls how long next step in state machine is going to wait. I'm planning to come up with a strategy of estimating wait time based on the filesize or some other criteria TBD. 
Renaming the variable as per your comment

Reply:
In general, if you're going to use string constants, I would suggest you create UPPER_CASE_NAMES for them, as long as any documentation to indicate where updates are necessary should a refactor happen (i.e., in the SFN definition).